model/registry/service.yaml

```diff
@@ -20,6 +20,13 @@ groups:
         type: string
         brief: >
           The version string of the service API or implementation. The format is not defined by these conventions.
+        note: |
+          When possible, SDKs MAY use language/platform-specific capabilities to
```

Comment:
This normative language belongs here: https://github.com/open-telemetry/semantic-conventions/blob/main/docs/resource/README.md#semantic-attributes-with-sdk-provided-default-value